### PR TITLE
Add conda-forge to installation instructions

### LINKED ISSUE #226: Package is also on conda-forge
Hello, @larsoner and I packaged pyRiemann for conda-forge: https://anaconda.org/conda-forge/pyriemann

If you're interested, the install docs should be updated.

Please LMK what you think!

Richard

### PR DESCRIPTION
**Pull request by hoechenberger**
Fixes #226

Also makes installation instructions consistent between readme and sphinx docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,21 +38,28 @@ or using pip+git for the latest version of the code :
 pip install git+https://github.com/pyRiemann/pyRiemann
 ```
 
-Anaconda is not currently supported, if you want to use anaconda, you need to create a virtual environment in anaconda,
-activate it and use the above command to install it.
+#### Using conda
+
+The package is distributed via [conda-forge](https://conda-forge.org).
+We recommend installing it into a new, dedicated environment.
```
Comment (agramfort) [inline README.md:44]: why a new env? then you end up with one env per package and it makes no sense.

I would say "to activate the environment" you are willing to update if it's not your base environment and then just conda install

my 2c

File: doc/installing.rst [hunk 2]
```diff
@@ -3,10 +3,14 @@
...
 ====================
 
-The easiest way to install a stable version of pyRiemann is through pypi, the python package manager :
+The easiest way to install a stable version of pyRiemann is through ``pip``, the Python package manager::
 
 ``pip install pyriemann``
 
+or via ``conda`` into a new, dedicated environment::
+
+``conda create --name=pyriemann --channel=conda-forge --override-channels pyriemann``
```
Comment (agramfort) [inline doc/installing.rst:12]: same remark

**Comment by hoechenberger** [comment]
Hello, thanks for your feedback, I will address your comments shortly!

File: README.md [hunk 3]
```diff
@@ -38,21 +38,28 @@ or using pip+git for the latest version of the code :
...
+The following command will create an environment named `pyriemann`.
```
Comment (sylvchev) [inline README.md:45]:
```suggestion
You could install it in your working environment, with the following command:
```

I agree, pyriemann do not need to install in a specific env.

**Review by qbarthelemy — APPROVED** [review]

**Comment by qbarthelemy** [comment]
Thx @hoechenberger !

**Pull request merged**